salt/_renderers/metalk8s_kubernetes.py

```diff
@@ -227,8 +267,16 @@ def render(yaml_data, saltenv='', sls='', **kwargs):
 
     data = yaml.load_all(yaml_data, Loader=salt.utils.yaml.SaltYamlSafeLoader)
 
+    objects = []
+    for obj in data:
+        if not obj:
+            continue
+        if isinstance(obj.get('items'), list):
```

Comment:
Did you really have to do this? IIUC, the idea is to push a `RoleBindingList`, which is supported by the K8s API: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#rolebindinglist-v1-rbac-authorization-k8s-io

In any case, I believe we'll want to replace this list with a `ClusterRoleBinding` instead.

Reply:
Oh and same for `ConfigMapList`: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#configmaplist-v1-core

Reply:
I did this because in the renderer we're generating the name/id of the state with the name & kind of the objet we want to create. When we've a *list kind, we don't have a name so there is nothing to create an unique name/id.

Reply:
My point was we could add the `*_list_[pre|ab]sent` states since they exist in the API, but I guess it's simpler this way :)